insights/parsers/qemu_xml.py

```diff
@@ -240,3 +237,88 @@ class VarQemuXML(BaseQemuXML):
         </domstatus>
     """
     pass
+
+
+class OpenStackInstanceXML(BaseQemuXML):
```

Comment:
This Parser didn't bond to any spec, it will not be able to get fired. what kind of spec do you want to run this parser? 


perhaps, this https://github.com/RedHatInsights/insights-core/pull/1920/ could help.

Reply:
@xiangce The spec was missed. Fixed is latest commit. Thanks!!

Reply:
It's not good to apply the same spec to both QemuXML and the new one OpenStackInstanceXML, better add some depentancies.

Reply:
@psachin I agree with @xiangce regarding the second parser for the same spec.  We normally have a 1:1 ratio of specs to parsers.  What makes this case different?

Reply:
@xiangce The parser will be fired only if `insights.components.openstack.IsOpenStackCompute` dependency is met.